api/src/parcel/get.handler.ts

```diff
@@ -0,0 +1,96 @@
+import { APIGatewayProxyResult } from 'aws-lambda';
+import * as AWS from 'aws-sdk';
+import { RDSDataService } from 'aws-sdk';
+import { ExecuteStatementRequest, SqlParametersList } from 'aws-sdk/clients/rdsdataservice';
+
+const CORS_HEADERS = {
+  'Access-Control-Allow-Headers': 'Content-Type',
+  'Access-Control-Allow-Origin': '*',
+  'Access-Control-Allow-Methods': 'OPTIONS,GET',
+};
+const SCHEMA = 'public';
+const SQL_QUERY = `SELECT address,
+                          public_lead_prediction,
+                          private_lead_prediction,
+                          geom
+                   FROM parcels
+                   WHERE geom && ST_SetSRID(ST_Point(:long, :lat), 4326)
+                   LIMIT 1`;
+
+async function getParcelData(
+  rdsService: RDSDataService,
+  params: SqlParametersList,
+): Promise<ParcelApiResponse> {
+  const executeParams: ExecuteStatementRequest = {
+    database: process.env.DATABASE_NAME ?? 'postgres',
+    resourceArn: process.env.RESOURCE_ARN ?? '',
+    schema: SCHEMA,
+    secretArn: process.env.CREDENTIALS_SECRET ?? '',
+    sql: SQL_QUERY,
+    parameters: params,
+  };
+  let body: ParcelApiResponse = {};
+
+  const results = await rdsService.executeStatement(executeParams).promise();
+  for (let record of results.records ?? []) {
+    body = {
```

Comment:
small error handling thing, but should we check that record has all values before indexing? like if record.length < 3 print an error? I can imagine an index out of bounds error here requiring a bit of digging.

Reply:
Done!